packages/common/src/search/types/IHubSearchOptions.ts

```diff
@@ -35,23 +35,24 @@ export interface IHubSearchOptions {
    * hubSearch
    */
   requestOptions?: IHubRequestOptions;
+  /**
+   * Bounding box for the search (NOT IMPLEMENTED)
+   */
   bbox?: string;
   /**
-   * Objects or fields to include
-   * e.g. `server.layers AS serverLayers`
+   * Objects or fields to include e.g. `server.layers AS serverLayers`
    */
   include?: string[];
   /**
-   * Enrichments
-   * Deprecated in favor of `include`
+   * DEPRECATED: Use `include` instead
    */
   enrichments?: Enrichments[];
   /**
    * Maximum number of results to return, per-page
    */
   num?: number;
   /**
-   * What page of results to return
+   * LEGACY. Use `start` and `num` instead
```

Comment:
For my edification, what's the difference between `legacy` and `deprecated` in this context.

Reply:
Good question - until we retire the v3 api, we can't kill LEGACY things. At least was my thinking at the time I wrote this